Framework/Algorithms/src/Multiply.cpp

```diff
@@ -56,6 +56,11 @@ void Multiply::setOutputUnits(const API::MatrixWorkspace_const_sptr lhs, const A
                               API::MatrixWorkspace_sptr out) {
   if (!lhs->isDistribution() || !rhs->isDistribution())
     out->setDistribution(false);
+
+  if (lhs->isRaggedWorkspace() && rhs->isRaggedWorkspace()) {
```

Comment:
I'm not sure this belongs. I'm getting a second opinion from Andrei

Reply:
Malcolm said it applies for multiplication and this fixed the behavior

Reply:
I spoke with Andrei and the changes to Multiply should be removed. Ragged and non-ragged workspaces should have the same behavior for multiplication. For divide, this PR is making ragged the same as non-ragged. I think for multiply, the two are the same before this PR.